pkg/nest/client.go

```diff
@@ -26,6 +33,12 @@ func Dial(rawURL string) (*Client, error) {
 	refreshToken := query.Get("refresh_token")
 	projectID := query.Get("project_id")
 	deviceID := query.Get("device_id")
+	protocols := strings.Split(query.Get("protocols"), ",")
```

Comment:
The array doesn't make sense. There is no priority check below. The first protocol specified will always be selected.